pkg/store/dirnames/dirnames.go

```diff
@@ -24,11 +25,22 @@ func LimaDir() (string, error) {
 		if err != nil {
 			return "", err
 		}
+		// on windows, DOS paths are needed by some tools like QEMU
```

Comment:
Could you explain DOS paths in the comment?

Reply:
I think the technical name would be 8.3 names

Reply:
`8dot3name`, e.g.

```console
C:\Users\Jan\Names>fsutil 8dot3name query
The registry state is: 2 (Per volume setting - the default).
```

Reply:
I hadn't expected to see them again, this century :-)